pep-0517.txt

```diff
@@ -166,48 +166,71 @@ The build backend object is expected to have attributes which provide
 some or all of the following hooks. The common ``config_settings``
 argument is described after the individual hooks::
 
-  def get_build_requires(config_settings):
+  def build_sdist(source_directory, sdist_directory, config_settings):
+      ...
+
+MUST build an unpacked sdist from the ``source_directory`` and place it into the
+specified ``sdist_directory``. If the ``source_directory`` is already an
+unpacked sdist (as opposed to a VCS source tree) then it SHOULD simply copy the
```

Comment:
- Do we specify how to tell what case it's handling?
- What about cases which are neither? E.g. if I download a tarball from Github and unpack it, it is neither a VCS checkout (no `.git` directory), nor an sdist. Is it OK to error out? Should frontend tools try something else if this fails?